freetextresponse/freetextresponse.py

```diff
@@ -279,6 +279,17 @@ def student_view(self, context={}):
             (Fragment): The HTML Fragment for this XBlock, which determines the
             general frame of the FreeTextResponse Question.
         """
+        student_id = self.get_student_id()
+        display_other_responses = self.display_other_student_responses
+        student_answer = self.student_answer
+        if student_answer and display_other_responses:
```

Comment:
Kind of a nit, but I find it to be a little safer to explicitly declare a default value, instead of relying on the `else` clause. It also reads a bit more cleanly like:

```python
other_responses = []
if student_answer and display_other_responses:
    other_responses = self.get_other_answers(student_id)
```